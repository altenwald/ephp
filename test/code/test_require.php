<?php

require_once __DIR__ . "/test_print_true.php" ;
require_once(__DIR__ . "/test_print_true.php");

require __DIR__ . "/test_print_true.php";
require(__DIR__ . "/test_print_true.php");

require("no_exists");
