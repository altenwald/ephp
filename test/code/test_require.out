1

1
1

1
1

1

Warning: require(no_exists): failed to open stream: No such file or directory in {{CWD}}/test/code/test_require.php on line 9

Fatal error: require(): Failed opening required 'no_exists' (include_path='.:') in {{CWD}}/test/code/test_require.php on line 9
